src/main/resources/application-local.yml

```diff
@@ -23,37 +23,42 @@ hmpps.sqs:
       queueName: cpr_court_cases_queue.fifo
       subscribeTopicId: courtcasestopic
       dlqName: cpr_court_cases_queue_dlq.fifo
-      dlqMaxReceiveCount: 1
+      dlqMaxReceiveCount: 2
```

Comment:
This is to match application-test so that the same localstack container can be used with both test and local running